manager/middlewares/rbac.go

```diff
@@ -5,9 +5,17 @@ import (
 	"app/base/utils"
 	"github.com/RedHatInsights/patchman-clients/rbac"
 	"github.com/gin-gonic/gin"
+	"github.com/prometheus/client_golang/prometheus"
 	"net/http"
+	"strconv"
 )
 
+var serviceErrorCnt = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
As it's intended also for other services it should be defined in middlewares/prometheus.go file.